examples/nextjs-nx/apps/app/specs/index.spec.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+
+import Index from '../pages/index';
+import preview from "jest-preview";
+
+describe('Index', () => {
+  it('should render successfully', () => {
+    const { baseElement } = render(<Index />);
+
+    preview.debug();
```

Comment:
Whether I make the test fail automatically or by using preview.debug(), I cannot get consistent auto-refresh on the jest-preview server

Reply:
Can you use
```js
 import {debug} from 'jest-preview'
...
debug();
...
``` 
instead? `preview.debug()` is kept mainly for a historical reason.